### ISSUE TITLE
not work in nwjs or electron

### ISSUE THREAD

**Issue by jonlepage** [minop1205/react-dnd-treeview#146]
**Describe the bug**
Hi, when i use nwjs or electron, the dnd of your lib not work!
Did you use something special thats can break the behavior in this kind of wrapper?
Is work fine if i run my app on heavy browser.
Dnd work with other libs.
thanks

**To Reproduce**
1. use nwjs or electron as wrapper
https://nwjs.io

**Expected behavior**
DND should work like in browser

**Screenshots**
![gjuuyyu](https://user-images.githubusercontent.com/24865815/192527262-ca187929-32b5-4e9c-ade9-06582f610973.gif)

**Desktop (please complete the following information):**
![image](https://user-images.githubusercontent.com/24865815/192528435-396d6690-3a36-42fe-bd64-9ee0e37883d0.png)

**Additional context**
the app is clean
no css or other stuff for the test
![image](https://user-images.githubusercontent.com/24865815/192528745-99f82fb5-672a-4f2b-bd36-26c3488907bc.png)

**Comment by minop1205** [comment]
@djmisterjon Thanks for the report.
I also tried to run it with nwjs and confirmed the same phenomenon.

I looked into it and found the following workaround.
https://github.com/nwjs/nw.js/issues/7531

I was able to work around the problem by adding the following line to package.json.

```
"chromium-args": "--mixed-context",
```

**Comment by jonlepage** [comment]
> @djmisterjon Thanks for the report. I also tried to run it with nwjs and confirmed the same phenomenon.
> 
> I looked into it and found the following workaround. [nwjs/nw.js#7531](https://github.com/nwjs/nw.js/issues/7531)
> 
> I was able to work around the problem by adding the following line to package.json.
> 
> ```
> "chromium-args": "--mixed-context",
> ```

interesting,nice catch it work!
But my issue now with is am not able to load 2 plugins in same time with arg `--mixed-context` !
```json
"chromium-args": "--mixed-context --disable-raf-throttling --max-old-space-size=8192 --enable-spell-checking --remote-debugging-port=9222 --force-gpu-mem-available-mb=9999999 --load-extension='./plugins/aamddddknhcagpehecnhphigffljadon','./plugins/fmkadmapgofadopljbjfkapdkoienihi'",
```

But i think is a parser issu from internal nwjs .
After some hours and many test, something defensively wrong!
![image](https://user-images.githubusercontent.com/24865815/192803113-c6479aa7-db22-4990-91d4-44a25b9041c2.png)


**I will report on the nwjs repo.**
Thank for your time and your workaround, your assomme.!
